src/index.ts

```diff
@@ -1,102 +1,119 @@
-const at = require('lodash/at')
-const isEmpty = require('lodash/isEmpty')
-import plural from 'plurals-cldr'
+const at = require('lodash/at');
+const isEmpty = require('lodash/isEmpty');
```

Comment:
This file is weirdly formatted now, with final `;` and 4-spaces indents, can you review that?